circle.yml

```diff
@@ -37,7 +37,8 @@ test:
     - cd $SRCDIR; ./bin/lint .
     - cd $SRCDIR; make client-test
     - cd $SRCDIR; make static
-    - cd $SRCDIR; make
+    - cd $SRCDIR; rm -f app/app probe/probe; GOOS=linux make
+    - cd $SRCDIR; rm -f app/app probe/probe; GOOS=darwin make
```

Comment:
Is this going to build a darwin executable (and stick it in the docker container, and upload it to the docker hub)?